iris_grib/tests/integration/load_convert/test_sample_file_loads.py

```diff
@@ -0,0 +1,223 @@
+# Copyright iris-grib contributors
+#
+# This file is part of iris-grib and is released under the LGPL license.
+# See COPYING and COPYING.LESSER in the root of the repository for full
+# licensing details.
+"""
+Integration tests for grib2 file loading.
+
+These load various files from iris-test-data, and compare the cube with a
+reference CML file, to catch any unexpected changes over time.
+
+"""
+
+# import iris_grib.tests first so that some things can be initialised
+# before importing anything else.
+import iris_grib.tests as tests
+
+from unittest import skipIf
+
+import iris
+
+# Skip out tests that fail because the file coding is not yet supported.
+# NOTE: now *only* used for the file 'GRIB/y_fastest/y_fast.grib2', as the
+# loader does not support X,Y dimensioned data (as opposed to Y,X).
+# TODO: fix this, or remove the test.
+skip_irisgrib_fails = skipIf(
+    True, "Current grib2 loader does not support this test(s)."
+)
+
+_RESULTDIR_PREFIX = ("integration", "load_convert", "sample_file_loads")
+
+
+@tests.skip_data
+class TestBasicLoad(tests.IrisGribTest):
+    def test_load_rotated(self):
+        cubes = iris.load(
+            tests.get_data_path(("GRIB", "rotated_uk", "uk_wrongparam.grib1"))
+        )
+        self.assertCML(cubes, _RESULTDIR_PREFIX + ("rotated.cml",))
+
+    def test_load_time_bound(self):
+        cubes = iris.load(
+            tests.get_data_path(("GRIB", "time_processed", "time_bound.grib1"))
+        )
+        self.assertCML(cubes, _RESULTDIR_PREFIX + ("time_bound_grib1.cml",))
+
+    def test_load_time_processed(self):
+        cubes = iris.load(
+            tests.get_data_path(("GRIB", "time_processed", "time_bound.grib2"))
+        )
+        self.assertCML(cubes, _RESULTDIR_PREFIX + ("time_bound_grib2.cml",))
+
+    def test_load_3_layer(self):
+        cubes = iris.load(
+            tests.get_data_path(("GRIB", "3_layer_viz", "3_layer.grib2"))
+        )
+        cubes = iris.cube.CubeList([cubes[1], cubes[0], cubes[2]])
+        self.assertCML(cubes, _RESULTDIR_PREFIX + ("3_layer.cml",))
+
+    def test_load_masked(self):
+        gribfile = tests.get_data_path(
+            ("GRIB", "missing_values", "missing_values.grib2")
+        )
+        cubes = iris.load(gribfile)
+        self.assertCML(cubes,
+                       _RESULTDIR_PREFIX + ("missing_values_grib2.cml",))
+
+    @skip_irisgrib_fails
+    def test_y_fastest(self):
```

Comment:
Why do you want to keep this test? We had already removed it from the iris repo: https://github.com/SciTools/iris/pull/3632

Reply:
Ok, I hadn't spotted that when I did this.

Reply:
done.